app/_includes/head.html

```diff
@@ -158,6 +158,14 @@
     href="/assets/hub.css?v={{ site.time | date: '%s' }}"
   />
   {% endif %}
+
+  {% assign latest_page_url = page.url | replace: page.kong_version, page.kong_latest.release %}
+  {% capture latest_page_exists %}{% page_exists {{latest_page_url}} %}{% endcapture %}
+
+  {% if page.kong_version == site.data.kong_latest.release and page.no_version != true %}<link rel="canonical" href="{{ site.links.web }}{{ page.url }}" />
+    {% elsif latest_page_exists == 'true' %}<link rel="canonical" href="{% capture latest_path %}{{ site.links.web }}{% assign latest_page_url = page.url | replace: page.kong_version, page.kong_latest.release %}{{ latest_page_url }}{% endcapture %}{{ latest_path }}" />
```

Comment:
```suggestion
    {% elsif latest_page_exists == 'true' %}<link rel="canonical" href="{% capture latest_path %}{{ site.links.web }}{{ latest_page_url }}{% endcapture %}{{ latest_path }}" />
```
Don't need the `assign` again since it's already assigned above.

Reply:
🤦‍♂️